src/search/cegar/utils_landmarks.cc

```diff
@@ -23,12 +23,9 @@ static FactPair get_fact(const LandmarkNode &node) {
 shared_ptr<LandmarkGraph> get_landmark_graph(const shared_ptr<AbstractTask> &task) {
     Options hm_opts;
     hm_opts.set<int>("m", 1);
```

Comment:
I'm not sure I fully understand the discussion here. This function just computes some kind of  "default" h^m landmarks, right? The previous code didn't use reasonable order and so doesn't the new code, so this is fine in my view.

Reply:
Btw., I don't know if this is Github's fault, but I see all comments regarding specific lines above the line instead of below the line. If I click on the plus next to a line I want to comment on, my comment will end up below the line. Did you do the same or did the comment end up above the line for another reason? I think it makes a lot of sense to have comments below the lines they refer to due to scrolling through the code from top to bottom...

Reply:
Then I think we should update the comment.